src/js/Views/RaceTrackView/components/OneTrack.js

```diff
@@ -0,0 +1,30 @@
+// 하나의 track을 가져야 함.
+// state를 받아 계속해서 element를 바꿔야하기 때문에 View로서 가지고 있는다.
+import { createArrowElement } from "./createArrowElement.js";
+import { createSpinElement } from "./createSpinElement.js";
+
+class OneTrack {
+  rootElement;
+  titleElement;
+
+  constructor(title) {
+    const rootElement = document.createElement('div');
+    rootElement.classList.add('mr-2');
+    this.rootElement = rootElement;
+
+    const titleElement = document.createElement('div');
+    titleElement.classList.add('car-player');
+    titleElement.textContent = title;
+    this.titleElement = titleElement;
+    rootElement.appendChild(this.titleElement);
+    rootElement.appendChild(createSpinElement());
```

Comment:
```suggestion
    const rootElement = createElement(`
      <div class="mr-2">
        <div class="car-player">
          ${title}
        </div>
        
        <div class="relative spinner-container">
          <span class="material spinner"></span>
        </div>
      </div>
    `);
```
제가 앞서 소개한 내용을 토대로 만들어본다면,
이 부분은 이렇게 표현할 수 있지 않을까요?
지금 불필요하게 복잡한 형태로 view가 표현된 것 같아요..!